### ISSUE TITLE
Postfixadmin password reset plugin not included as of the latest version

### ISSUE THREAD

**Issue by belldandu** [RainLoop/rainloop-webmail#1201]
### RainLoop version, browser, OS:
1.10.4.183, Firefox, Android Marshmallow (6.0)
### Expected behavior and actual behavior:
Postfixadmin password reset plugin should be installable from the packages list.

Postfixadmin password reset plugin is not in the packages list.
### Steps to reproduce the problem:
Goto the packages list in the admin panel for rainloop.

**Comment by zilon84** [comment]
Same for me. I am using RainLoop from the ports collection of FreeBSD.

**Comment by Darlelet** [comment]
Indeed, doesn't seem to be available from the supported packages in rainloop's latest version. Meanwhile your users should be able to change their password from 'PostfixadminRootURL'/users where they'll be asked for their "email address / password" combination in order to access their mailbox settings (password and alias)

**Comment by zilon84** [comment]
Still not included in the latest package. Is there a way to manually activate the plugin?

**Comment by navossoc** [comment]
Same... where is the plugin on the package list?

It is not included here:
http://repository.rainloop.net/v1/repository.json
